stf/src/sgx.rs

```diff
@@ -96,33 +86,35 @@ impl Stf {
                         reserved_balance,
                     )
                     .dispatch(sgx_runtime::Origin::ROOT)
+                    .map_err(|_| StfError::Dispatch)?;
+                    Ok(())
                 }
                 TrustedCall::balance_transfer(from, to, value) => {
-                    //FIXME: here would be a good place to really verify a signature
```

Comment:
This one can be removed I believe, as we check the `TrustedCallSigned` signature outside?

@brenzi what do you think?